state-sync/state-sync-driver/src/bootstrapper.rs

```diff
@@ -83,7 +83,12 @@ impl VerifiedEpochStates {
     }
 
     /// Sets `verified_waypoint` to true
-    pub fn set_verified_waypoint(&mut self) {
+    pub fn set_verified_waypoint(&mut self, waypoint_version: Version) {
+        info!(LogSchema::new(LogEntry::Bootstrapper).message(&format!(
+            "The waypoint has been verified! Waypoint version: {:?}.",
```

Comment:
nit: don't have to debug print?

```suggestion
            "The waypoint has been verified! Waypoint version: {}.",
```